### DISCUSSION TITLE
Please tell me how to set up multiple IPs for Detection TUNING in 2.4.70

### DISCUSSION THREAD

**Discussion by larry1788** [Security-Onion-Solutions/securityonion#13186] [Unsupported Versions] [answered]
Hi All
    Please tell me how to set up multiple IPs for Detection TUNING in 2.4.70
![多IP問題-1](https://github.com/Security-Onion-Solutions/securityonion/assets/69907736/592329ef-871e-4d3b-9009-682c98432af7)

thank you for your help

**Comment by larry1788** [comment]
Just tested the results
Only one CIDR can be added at a time. After adding, you need to wait 15 minutes before adding another parameter.

**Reply by izidood** [reply, marked as answer]
Have you tried to just reload the page and add another CIDR.
Thats what iv'e done. Added a bunch with no problems, if refreshing the site.

**Reply by larry1788** [reply]
Thanks for the information, this method works!

**Reply by izidood** [reply]
btw, the refresh has to be done everytime you do something on the same instance. fx if you also need to add notes etc.
It's a bit messy but it works atleast.

**Reply by defensivedepth** [reply]
Yes, the refresh issue is a known bug, it will be fixed in the next release.
